sdk/mongocluster/Azure.ResourceManager.MongoCluster/api/Azure.ResourceManager.MongoCluster.netstandard2.0.cs

```diff
@@ -0,0 +1,707 @@
+namespace Azure.ResourceManager.MongoCluster
+{
+    public partial class FirewallRuleCollection : Azure.ResourceManager.ArmCollection, System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.MongoCluster.FirewallRuleResource>, System.Collections.Generic.IEnumerable<Azure.ResourceManager.MongoCluster.FirewallRuleResource>, System.Collections.IEnumerable
```

Comment:
I think network RP has something similar, maybe we should rename this resource?

Reply:
renamed